lib/src/main/java/ECSE428Project/service/LogoutService.java

```diff
@@ -19,29 +19,42 @@
     private AccountRepository accountRepository;
 
 
-    //Do we need to implement a "find current logged in account" method?
+    //It would be good to implement a "find current logged in account" method
```

Comment:
This is done in the frontend using localStorage but it would still be a good idea to have that method